setup.py

```diff
@@ -21,6 +21,7 @@
         "jinja2",
     ],
     "delayed": [],  # keeping for backwards compatibility
+    "pyodide": ["ipycytoscape", "jinja2"],
```

Comment:
It's mostly for convenience, but it's a bit more annoying in pyodide, since *user* code would need to install these extras, rather than whoever is setting up the environment.

Easy to remove, however.